build.xml

```diff
@@ -76,6 +76,13 @@
             <arg value="-B${workdir}/cpp/cmake-build"/>
             <arg value="-DIDL_BASE=${workdir}/idl/install"/>
         </exec>
+        <exec executable="sed" failonerror="true" failifexecutionfails="true">
+            <arg value="-i"/>
+            <arg value="1s/5\.0\.1/5.0.2/;18s/5:1:0/5:2:0/"/>
```

Comment:
@mliszcz, I am not an expert in the usage of sed and I'm curious...
What is the advantage of adding '1' and '18' in front of 's/' on this line?
What does it do?

Reply:
The number (1 or 18) shows the lines for which the operation (s=substitute) applies. I just wanted to prevent accidental modifications from other lines (just in case "5.0.1" would be used in a different context).

Please check: https://www.gnu.org/software/sed/manual/sed.html#sed-script-overview
```
[addr]X[options]
```
> X is a single-letter sed command. [addr] is an optional line address. If [addr] is specified, the command X will be executed only on the matched lines. [addr] can be a single line number, a regular expression, or a range of lines (see sed addresses).

Reply:
Thanks for the explanations.